### ISSUE TITLE
CRISP: Integrate coarse jaws

### ISSUE THREAD

**Issue by DominicOram** [ISISComputingGroup/IBEX#4303]
As an instrument scientist on CRISP I would like to be able to control my coarse jaws through IBEX. The jaws are beckhoff controlled but are currently not communicating with IBEX in any way. This ticket will most likely use tcIoc to communicate with the TwinCat collaboration code running on the device.

**Comment by DominicOram** [comment]
Main PR at https://github.com/ISISComputingGroup/EPICS-tcIoc/pull/4

**Comment by Tom-Willemsen** [comment]
Mostly good, a few relatively minor comments in code but once those are resolved I'm happy to merge as this functionally works as far as I can tell.

**Comment by DominicOram** [comment]
Just remembered that I need to make an OPI for this, sorry.

**Comment by DominicOram** [comment]
With changes in https://github.com/ISISComputingGroup/EPICS-jaws/pull/9 this should be controllable with the normal jaws OPI
